### PR TITLE
chore: apply code standards in util context

### PR DESCRIPTION
**Pull request by orlandocavassani**
### Description

Apply code standards in util context

### CODE REVIEW

File: src/Util/CacheComponent.php [hunk 1]
```diff
@@ -29,21 +31,23 @@ class CacheComponent implements CacheComponentInterface
      *
      * @return mixed
      */
-    public function get(string $key)
+    public function get(string $key): mixed
```
Comment (ezandonai) [inline src/Util/CacheComponent.php:32]: You can remove phpdoc return.

File: src/Util/CacheComponentInterface.php [hunk 2]
```diff
@@ -15,7 +15,7 @@ interface CacheComponentInterface
      *
      * @return mixed
      */
-    public function get(string $key);
+    public function get(string $key): mixed;
```
Comment (ezandonai) [inline src/Util/CacheComponentInterface.php:16]: You can remove phpdoc return.

**Review by ezandonai — APPROVED** [review]

File: src/Util/CacheComponent.php (hunk 1 above)
Comment (orlandocavassani) [reply src/Util/CacheComponent.php:32]: done

File: src/Util/CacheComponentInterface.php (hunk 2 above)
Comment (orlandocavassani) [reply src/Util/CacheComponentInterface.php:16]: done

**Pull request merged**